app/views/declaration/destinationCountries/country_of_routing.scala.html

```diff
@@ -42,17 +42,14 @@
     @helper.form(routes.RoutingCountriesController.submitRoutingCountry(mode), 'autoComplete -> "off") {
         @helper.CSRF.formField
 
-        @*TODO move id to field_autocomplete*@
-        <div id="origination-country-field">
-            @components.fields.field_autocomplete(
-                form("country"),
-                "",
-                None,
-                None,
-                messages(s"declaration.${page.id}.empty"),
-                AutoCompleteItem.fromCountry(Countries.allCountries, _.countryCode)
-            )
-        </div>
+        @components.fields.field_autocomplete(
```

Comment:
Was the div with id not useful for tests?

Reply:
I'm checking this now, if yes, I'll fix tests. Unit tests don't use it

Reply:
`field_autocomplete` has his own id based on the field name, so this should be enough